tests/vmi_multus_test.go

```diff
@@ -1029,6 +1042,19 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 	return setUpInterface(vmi, interfaceName)
 }
 
+func configSRIOVInterface(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) error {
+	if vmi == nil {
+		return fmt.Errorf("the VMI must exist for us to configure an interface in it")
+	}
+
+	interfaceName := getSRIOVInterfaceNameByMAC(virtClient, vmi, interfaceMac)
+	if interfaceName == "" {
+		return fmt.Errorf("no SRIOV interface on vmi %s with mac: %s", vmi.Name, interfaceMac)
```

Comment:
I am not sure I understand what you meant here. we check if the interface name is empty string. for the mac, we check if it matches one of the interfaces in the vmi [here](https://github.com/kubevirt/kubevirt/blob/3730054890aae6b1d0ed2d5f17fa77ce58ce79ac/tests/vmi_multus_test.go#L1113) 

if I will changed it to:  "could not configure interface: no SRIOV interface on vmi %s with mac: %s" 
would that be OK?

Reply:
Sorry for the confusion, I did not got it right
You could change it to `fmt.Errorf("Could not configure SRIOV interface, VMI '%s' does not have interface with the MAC address '%s'", vmi.Name, macAddress)`

BTW, It could be easier and more elegant (IMO) if `getSRIOVInterfaceNameByMAC` would return an error we could warp like I suggested [here](https://github.com/kubevirt/kubevirt/pull/4001/files#r513422504)
then it would look like 
```
configureSRIOVInterfaceStaticIPByMAC() {
    ...
    interfaceName := getSRIOVInterfaceNameByMAC(..)
    if err != nil {
        fmr.Errorf("Could not configure SRIOV interface with MAC '%s': %v, macAddress, err)
    }
    ...
}
```

Reply:
changed configureSRIOVInterfaceStaticIPByMAC to return an error!